client/src/Components/Common/Card/index.js

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Card } from 'antd';
+import Rating from '../Rating';
+
+import './style.css';
+
+const { Meta } = Card;
+
+const CardComponent = ({
+  title,
+  rating,
+  reviewersNo,
+  location,
+  price,
+  cover,
+}) => (
+  <Card
+    hoverable
+    className="card"
+    style={{ width: 300 }}
+    cover={
```

Comment:
you can but the width inside CSS file instead here

Reply:
I put it here because the inline style takes priority , when I put it as you said it doesn't take the style although I put !important .

Reply:
yeah, so just keep inline-style.

Reply:
If you reached the element correctly you wouldn't even need to use `!important`